### PR TITLE
Pyne dependency updates from Tues hackathon

### PR DESCRIPTION
**Pull request by bam241**
working version for me

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,24 +1,25 @@
...
+#{% set variant = "openblas" %}
 
 package:
   name: pyne
   version: {{ version }}
 
 source:
   fn: pyne-{{ version }}-{{ sha256 }}.tar.gz
-  url: https://github.com/{{ org }}/pyne/archive/{{ version }}.tar.gz
+  url: https://github.com/pyne/pyne/archive/0.7.0-rc1.tar.gz
```
Comment (opotowsky) [inline recipe/meta.yaml]: You can keep the org and version variables in here for consistency, but would just have to add the '-rc1' at the end

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,24 +1,25 @@
...
   sha256: {{ sha256 }}
 
 build:
   number: {{ build }}
   skip: True  # [win or (osx and py34)]
   detect_binary_files_with_prefix: true
-  features:
-    - blas_{{ variant }}
+  #features:
+  # - blas_{{ variant }}
```
Comment (opotowsky) [inline recipe/meta.yaml]: These can probably be deleted

File: recipe/meta.yaml [hunk 3]
```diff
@@ -1,24 +1,25 @@
...
 
 package:
   name: pyne
   version: {{ version }}
 
 source:
   fn: pyne-{{ version }}-{{ sha256 }}.tar.gz
-  url: https://github.com/{{ org }}/pyne/archive/{{ version }}.tar.gz
+  url: https://github.com/pyne/pyne/archive/0.7.0-rc1.tar.gz
+  #  url: https://github.com/{{ org }}/pyne/archive/{{ version }}.tar.gz
```
Comment (kkiesling) [inline recipe/meta.yaml]: delete line

File: recipe/meta.yaml [hunk 4]
```diff
@@ -29,37 +30,21 @@ requirements:
     - {{ compiler("fortran") }}
   host:
     - python
-    - future
     - setuptools
```
Comment (opotowsky) [inline recipe/meta.yaml]: can be deleted

File: recipe/meta.yaml (hunk 1 above)
Comment (kkiesling) [inline recipe/meta.yaml]: should we use the org and version variables above for in this url?

File: recipe/meta.yaml [hunk 5]
```diff
@@ -1,24 +1,25 @@
 {% set org = "pyne" %}
-{% set version = "0.5.11" %}
+{% set version = "0.7.0" %}
 {% set build = "0" %}
-{% set sha256 = "ba05917b0de446ec520857655e77967a783e3bc723299faf5989f532d6b0647a" %}
-{% set variant = "openblas" %}
+{% set sha256 = "19e104a05aa56ef2e747f5e4a1c4cffefdb57a592a300823265ba72a1b55d4f3" %}
+#{% set variant = "openblas" %}
```
Comment (kkiesling) [inline recipe/meta.yaml]: delete

File: recipe/meta.yaml (hunk 2 above)
Comment (kkiesling) [inline recipe/meta.yaml]: delete

File: recipe/meta.yaml [hunk 6]
```diff
@@ -29,37 +30,21 @@ requirements:
...
   run:
     - python
-    - future
-    - blas 1.1 {{ variant }}
-    - openblas 0.2.20|0.2.20.*
-    - lapack
-    - hdf5 1.10.1
-    - bzip2 1.0.*
-    - xz 5.2.*
-    - numpy
```
Comment (opotowsky) [inline recipe/meta.yaml:53]: might want to reconsider?

File: recipe/meta.yaml [hunk 7]
```diff
@@ -29,37 +30,21 @@ requirements:
...
-    - scipy
+    - hdf5 <1.10.5
```
Comment (kkiesling) [inline recipe/meta.yaml]: we may want this to only have the version requirement if using python 3.7

File: recipe/build.sh [hunk 8]
```diff
@@ -6,7 +6,7 @@ if [ "$(uname)" == "Darwin" ]; then
 else
   skiprpath=""
 fi
-export FC=gfortran
+#export FC=gfortran
```
Comment (kkiesling) [inline recipe/build.sh:9]: delete

**Review by kkiesling — CHANGES_REQUESTED** [review]
just some clean up of lines that were commented out

File: recipe/meta.yaml [hunk 9]
```diff
@@ -29,37 +26,21 @@ requirements:
...
+    - future
     - jinja2
-
 test:
   requires:
-    - setuptools
     - nose
-    - pytables
   source_files:
     - pyne
```
Comment (opotowsky) [inline recipe/meta.yaml:45]: @kkiesling and I have determined this is not needed for python 2 or 3.6

Comment (opotowsky) [reply recipe/meta.yaml:45]: (only src and tests are needed)

**Review by kkiesling — APPROVED** [review]
All checks pass! :tada: !!

**Comment by opotowsky** [comment]
:fireworks: :yellow_heart: :+1: :smiley: :smiley_cat: @kkiesling @bam241 :exclamation: :tada:

**Pull request merged**